modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -5,20 +5,72 @@ This module is included in the following assemblies:
 
 :_mod-docs-content-type: PROCEDURE
 [id="ossm-configuring-distr-tracing-tempo_{context}"]
-= Configuring the distributed tracing platform (Tempo)
+= Configuring the {TempoName} and the {OTELName}
 
-You can expose tracing data to the {TempoName} stack by appending a named element and the `zipkin` provider to the `spec.meshConfig.extensionProviders` specification in the `ServiceMehControlPlane`, as shown in the following example. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the Tempo distributor service endpoint.
+You can expose tracing data to the {TempoName} by appending a named element and the `opentelemetry` provider to the `spec.meshConfig.extensionProviders` specification in the `ServiceMeshControlPlane`. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the OpenTelemetry Collector endpoint.
 
-You can create a TempoStack instance in the `tracing-system` namespace _after_ creating the `ServiceMeshControlPlane` and the `ServiceMeshMemberRoll` resources.
+//As of July 2, 2024, there is no common attribute for OpenTelemetry Collector.
+
+You can create a {OTELName} instance in a mesh namespace and configure it to send tracing data to a tracing platform backend service.
 
 .Prerequisites
 
-* You have installed the {TempoOperator} and {SMProductName} Operator in the `openshift-operators` namespace.
-* You have created namespaces such as `istio-system` and `tracing-system`.
+* You have created a TempoStack instance using the Red Hat {TempoOperator} in the `tracing-system` namespace. For more information, see "Installing {TempoName}" in the "Additional resources" section.
+
+* You have installed the {OTELOperator} in either the recommended namespace or the namespace `openshift-operators`. For more information, see "Installing the {OTELName}" in the "Additional resources" section.
+
+* If using Service Mesh 2.5 or earlier, set `spec.tracing.type` to `None` so tracing data can be sent to the OpenTelemetry Collector.
 
 .Procedure
 
-. Configure the `ServiceMeshControlPlane` resource to define an extension provider:
+. Create an OpenTelemetry Collector instance in a mesh namespace. This example uses the `bookinfo` namespace:
++
+[source, yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: bookinfo  # <1>
+  annotations:
+    sidecar.istio.io/inject: 'true' # <2>
```

Comment:
Does the valid value have to carry the single quotes? I was a little confused by this because the "true" value for [true](https://github.com/openshift/openshift-docs/pull/77583/files#diff-fd2c28f3a14abe8d80f09838366315d2c7b5f44cc550ee9095e2a3ca9dcb1d28R49) below is plain text with no markers.